pylintrc

```diff
@@ -326,7 +326,7 @@ max-attributes=7
 min-public-methods=2
 
 # Maximum number of public methods for a class (see R0904).
-max-public-methods=22
+max-public-methods=25
```

Comment:
@mtyaka Yup, I know you're not causing the issue here - but when we encounter this type of situation, if we just disable pylint we are just leaving the issue for later; it still usually shows a problem - I would like to stick to the pylint rules, and make the necessary changes to get pylint happy rather than sliencing it. Sometimes it feels silly, but it makes it easier to notice/check for real issues when we disable less warnings, and it helps maintaining a more consistent code style.

In this case, just rebasing after #63 is merged might be all that is needed, but I still wanted to mention it, and ensure we don't bump this more than it already is.

On the topic of pylint, see also OC-1572, to address another part of the debt we have on pylint warning disabling.

CC @open-craft/core FYI, since that's something I mentioned in a few PRs already.